README.tfdoc.hcl

```diff
@@ -93,145 +90,217 @@ section {
     END
 
     section {
-      title = "Main Resource Configuration"
+      title = "Top-level Arguments"
```

Comment:
According to the template this should be two indented sections:

`Module Argument Reference`
- `Main Resource Configuration`